Dev/config/strings.js

```diff
@@ -9,6 +9,7 @@ export default {
     TotalAssignments: 'Total Assignments',
     Ready: 'Ready',
     NotReady: 'Not Ready',
+    ClassAlreadyJoined: 'You\'re already in this class',
```

Comment:
What do you think of adding some fun personality messaging here? something like: "You have already joined this class earlier."
and then in a new line and smaller font: "We know it is exciting to be in this class. We are excited to have you the class too :)"

thoughts?